website/docs/intro/install/install-opentofu.sh

```diff
@@ -558,10 +558,17 @@ EOF
 # This function installs OpenTofu via the yum command line utility. It returns $TOFU_INSTALL_EXIT_CODE_INSTALL_REQUIREMENTS_NOT_MET
 # if yum is not available.
 install_yum() {
-  if ! command_exists "yum"; then
+  RPM_PACKAGE_MANAGER=""
```

Comment:
This file actually lives in the get.opentofu.org repository and needs to be changed there instead.  It was accidentally committed to the main repository and removed in https://github.com/opentofu/opentofu/pull/1314.

Reply:
What do you think about creating a separate command for yum and dnf?